docs/source/tutorials/package.rst

```diff
@@ -1,22 +0,0 @@
-.. _lightly-tutorials: 
-
-Python Package
-===================================
-
-With the Lightly SSL framework you can use the power of self-supervised learning
```

Comment:
Q: would it be possible to keep this file (and info) but have the tutorials unfolded in the navigation? Unless you find the info about pytorch and lighting irrelevant, its good for me

Reply:
I actually tried this and the only way I could get it working was by adding it as a "Tutorial" page. But I found this a bit confusing. I would also consider the information on this page not crucial, from the package and the tutorials it is pretty clear that lightly is built on pytorch (lightning), and the `toctree` is now part of the main tree so this information is not lost.